specification/cognitiveservices/data-plane/Language/preview/2022-05-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,2383 @@
+      "x-ms-discriminator-value": "NumericRange",
+      "properties": {
+        "rangeKind": {
+          "type": "string",
+          "enum": [
+            "Number",
+            "Speed",
+            "Weight",
+            "Length",
+            "Volume",
+            "Area",
+            "Age",
+            "Information",
+            "Temperature",
+            "Currency"
+          ],
+          "x-ms-enum": {
+            "name": "RangeKind",
+            "modelAsString": true
+          },
+          "description": "The kind of range that the resolution object represents."
+        },
+        "minimum": {
+          "type": "number",
+          "format": "double",
+          "description": "The beginning value of  the interval."
+        },
+        "maximum": {
+          "type": "number",
+          "format": "double",
+          "description": "The ending value of the interval."
+        }
+      },
+      "required": [
+        "rangeKind",
+        "minimum",
+        "maximum"
+      ]
+    },
+    "TemporalModifier": {
+      "type": "string",
+      "description": "An optional modifier of a date/time instance.",
+      "enum": [
+        "AfterApprox",
+        "Before",
+        "BeforeStart",
+        "Approx",
+        "ReferenceUndefined",
+        "SinceEnd",
+        "AfterMid",
+        "Start",
+        "After",
+        "BeforeEnd",
+        "Until",
+        "End",
+        "Less",
+        "Since",
+        "AfterStart",
+        "BeforeApprox",
+        "Mid",
+        "More"
+      ],
+      "x-ms-enum": {
+        "name": "Modifier",
+        "modelAsString": true
+      }
+    },
+    "TimeExpression": {
+      "type": "string",
+      "description": "An extended ISO 8601 date/time representation as described in (https://github.com/Microsoft/Recognizers-Text/blob/master/Patterns/English/English-DateTime.yaml)"
+    },
+    "LUISTargetIntentResult": {
+      "type": "object",
+      "description": "It is a wrap up of LUIS Generally Available response.",
+      "x-ms-discriminator-value": "luis",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TargetIntentResult"
+        }
+      ],
+      "properties": {
+        "result": {
+          "type": "object",
+          "description": "The actual response from a LUIS Generally Available application."
+        }
+      }
+    },
+    "QuestionAnsweringTargetIntentResult": {
+      "type": "object",
+      "description": "It is a wrap up a Question Answering KB response.",
+      "x-ms-discriminator-value": "question_answering",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TargetIntentResult"
+        }
+      ],
+      "properties": {
+        "result": {
+          "type": "object",
+          "$ref": "../2021-07-15-preview/questionanswering.json#/definitions/KnowledgeBaseAnswers",
+          "description": "The generated answer by a Question Answering KB."
+        }
+      }
+    },
+    "AnalyzeConversationJobsInput": {
```

Comment:
We should probably pull in changes to the analyzeconversations.json spec from the recently merged 2022-03-01-preview updates.

I can help with this today by creating a separate PR against the feature branch, which we can merge first.

Reply:
I have just created this PR to update the feature branch to include changes from March-1, so hopefully this will make the diff much smaller and will include any recent updates in this file.

https://github.com/Azure/azure-rest-api-specs/pull/18580